modules/dnn/src/int8layers/quantization_utils.cpp

```diff
@@ -15,7 +15,10 @@ namespace dnn
 static void broadcast1D2TargetMat(Mat& data, const MatShape& targetShape, int axis)
```

Comment:
You can use InputArray for data and get rid of copy before the call.

Reply:
In this case, I do need a copy. The input `std::vector` must be copied into the matrices `scalesMat` and `zeropointsMat` that will be used to compute the quantization in the `forward` function.